canonicalwebteam/search/views.py

```diff
@@ -24,7 +24,7 @@ def build_search_view(
     template_path="search.html",
     search_engine_id="009048213575199080868:i3zoqdwqk8o",
     site_restricted_search=False,
-    request_limit="2000/day",
+    request_limit="2000/day;6/minute;1/second",
```

Comment:
I still don't get it :) Maybe it's worth adding TODO comment here as well so that there is no confusion for others too.
So above in the README you're saying `Defaults to 2/second, 100/minute, 2000/day`. But here it has a different value by default.

Reply:
The value "2000/day;6/minute;1/second" is for testing purposes, as to qa effectively you need to reach the rate limit and see the corresponding error:
![image](https://github.com/user-attachments/assets/21ccc598-9f7e-49f3-ba6c-63aa08d7f8fe)
This wouldn't be practical if we set it to the production limit of "2000/day;100/minute;2/second".
If you follow the QA instructions, and you reach the rate limit that is _currently_ set, you can give it a qa +1, as it is working as intended. I think the todo section in the description is sufficient enough context.
If it makes it easier we can split the review into qa and code. wherein you review the code after the rate limit has been adjusted to the production limit.